### PR TITLE
OSDOCS-6049: Documented an external load balancer for vSphere note

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-6049](https://issues.redhat.com/browse/OSDOCS-6049)

Version(s):
4.13

Link to docs preview:
[External load balancers that supports multiple vSphere subnets](http://file.emea.redhat.com/dfitzmau/OSDOCS-6049/release_notes/ocp-4-13-release-notes.html#ocp-4-13-installation-and-update)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -104,6 +104,20 @@ After you run the installation program for {product-title} on vSphere, the defau
 
 For more information, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc#configuring-vsphere-regions-zones_installing-vsphere-installer-provisioned-customizations[Configuring regions and zones for a VMware vCenter].
 
+[id="ocp-4-13-installation-vsphere-external-lb-multisubnets"]
+==== External load balancers that supports multiple vSphere subnets
```
Comment (GroceryBoyJr) [inline release_notes/ocp-4-13-release-notes.adoc:108]:
```suggestion
==== External load balancers that support multiple vSphere subnets
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -104,6 +104,20 @@ After you run the installation program for {product-title} on vSphere, the defau
...
+
+You can configure an {product-title} cluster to use an external load balancer that supports multiple subnets. If you use multiple subnets, you can explicitly list all the IP addresses in any networks that are used by your load balancer targets. This configuration can reduce maintenance overhead because you can create and destroy nodes within those networks without reconfiguring the load balancer targets.
+
+For more information, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#nw-osp-configuring-external-load-balancer_installing-vsphere-installer-provisioned[Configuring an external load balancer].
+
+[id="ocp-4-13-installation-vsphere-encrypt-vm"]
+==== Support for encrypting a VM before installing a cluster on VMware vSphere
+
+In {product-title} {product-version}, you can encrypt your virtual machines before you install a cluster on VMware vSphere with user-provisioned infrastructure.
```
Comment (GroceryBoyJr) [inline release_notes/ocp-4-13-release-notes.adoc:117]:
```suggestion
In an {product-title} {product-version}, you can encrypt your virtual machines before you install a cluster on VMware vSphere with user-provisioned infrastructure.
```

**Review by GroceryBoyJr — COMMENTED** [review]
Good work @dfitzmau - two small nits please.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (dfitzmau) [reply release_notes/ocp-4-13-release-notes.adoc:117]: Changed to "For OCP 4.13".

**Pull request merged**